spec/lib/embed/viewer/file_spec.rb

```diff
@@ -48,6 +48,40 @@
     end
   end
 
+  describe 'message' do
```

Comment:
Should these tests go into authorization_spec.rb?  It seems like these tests are about the logic in Authorization.